receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_constants.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpuscraper
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/consumer/pdata"
+)
+
+// cpu metric constants
+
+var (
+	StateLabel       = "state"
+	CPULabel         = "cpu"
+	ProcessNameLabel = "process_name"
+)
+
+var (
+	UserStateLabelValue      = "user"
+	SystemStateLabelValue    = "system"
+	IdleStateLabelValue      = "idle"
+	InterruptStateLabelValue = "interrupt"
+	IowaitStateLabelValue    = "iowait"
+)
+
+func InitializeMetricCPUSecondsDescriptor(descriptor pdata.MetricDescriptor) pdata.MetricDescriptor {
```

Comment:
File an issue to provide a `CopyTo` for all the generated objects so you can construct a MetricDescriptor then call just `CopyTo`.

Reply:
Issue created

Reply:
This is updated to use `CopyTo` now